### DISCUSSION TITLE
MacBook Pro 13,2 Touch-ID

### DISCUSSION THREAD

**Discussion by Samir-eus** [community/community#37788] [Other Feature Feedback, Questions, & Ideas]
I install macOS Ventura on MacBook Pro 13,2 by OpenCore Legacy Patcher 5.0 and everything fine else Touch-ID doesn’t appear in system setting **please note than my touch-id is work perfect in macOS 12** but in in macOS 13 with OpenCode dons't work

**Comment by jfullstackdev** [comment]
there are so many possible instances why it's not working, 

you may want to check this out

https://dortania.github.io/OpenCore-Legacy-Patcher/MODELS.html

**Reply by djpetere** [reply]
I'm another user who has a problem with MBP 13.2 and the invisible touch id option.
It doesn't matter if it's a pure OPLP 0.5.1 ventura installation or an update. the toich id option is just gone.

the above answer is pointless, it adds nothing to the solution of the problem.

**Reply by Samir-eus** [reply]
Nothing new just waiting 🥹🥹🥹

**Reply by Samir-eus** [reply]
I tray everything new install, update but it still

**Reply by nachogargar** [reply]
Same problem here guys 😭 I have not found any solution... Hopefully a new version of OPLP will be released soon fixing the problem!

**Reply by djpetere** [reply]
OCLP update has arrived, unfortunately touch id still don't work

**Reply by chkpnt** [reply]
I'm also experiencing issues with Touch-ID on my MacBookPro13,2. But in my case, Touch-ID still appears in the system settings, but the system says Touch-ID is locked.  And I'm unable to register a new finger:

<img width="782" alt="grafik" src="https://user-images.githubusercontent.com/1956979/224564193-97105600-c3c6-4024-b107-3b8cd51cf7d2.png">

**Reply by djpetere** [reply]
For versions older than Sonoma, I found a solution to make the touch ID option visible again.

in the opencore legacypatcher settings, in the tab: settings-SMbios-smbios spoof level, and select "minimal".

the rest remains unchanged, just a minor change and it turns on touch ID.


unfortunately, in the OCLP version for Sonoma, the touch-id is visible but unfortunately it does not work (touch ID visibility is available if the "smbios spoof level" is set to minimum and it is necessary to select the smbios spoof model for the system to start, unfortunately when you try to change the settings it asks for the administrator password . (despite the correct password, it is not possible to modify the settings.)

**Reply by riceee1** [reply]
does anyone know a fix yet for Touch ID on Sonoma?

**Reply by dpmaycry** [reply]
> settings.)

Lots of thanks; Don't forget to change SIMBIOS Spoof Model to your current model as for me is MacBookPro13,2; Got my TouchID back on Sequoia with OCLP 2.2.0.

**Reply by Kimina-Santuri** [reply]
Hey just wanted to pop in and ask anyone who goes through this, Do not try this unless you are super sure about it, found myself in a weird loop where the computer wouldn't boot and running OCLP in safe mode wouldn't actually fix it because it thought the computer model was different than what it was. So no matter how I kept building opencore to the target disk it wouldn't work.

Couldn't find a workaround, hard lesson learnt!

**Comment by alongwor** [comment]
> For versions older than Sonoma, I found a solution to make the touch ID option visible again.
> 
> in the opencore legacypatcher settings, in the tab: settings-SMbios-smbios spoof level, and select "minimal".
> 
> the rest remains unchanged, just a minor change and it turns on touch ID.
> 
> unfortunately, in the OCLP version for Sonoma, the touch-id is visible but unfortunately it does not work (touch ID visibility is available if the "smbios spoof level" is set to minimum and it is necessary to select the smbios spoof model for the system to start, unfortunately when you try to change the settings it asks for the administrator password . (despite the correct password, it is not possible to modify the settings.)

I have exactly the same situation. Following with interest :-)

**Comment by alongwor** [comment]
OC Legacy Pater v1.1.0 landed with me today and seems to have fixed the issue for me - I can now unlock with touchid

**Reply by ederlopesmoraes** [reply]
what did you do?

**Reply by djpetere** [reply]
The latest version of OCLP fixes the problem. I also have touchid options available, but it is important to change the settings to settings-SMbios-smbios spoof level, and select "minimal".
